docs/InspectingContracts.md

```diff
@@ -0,0 +1,140 @@
+# Inspecting Terra Contracts
+
+Contract schemas for Terra can be found in the [contracts](/contracts) folder. Each contract in this directory has a subdirectory called **schema**. In these folders, you can learn how to query, execute, and instantiate each contract. Queries are located in the **query_msg.json** files.
+
+There are two types of queries: string queries and object queries. String queries are used to read a property of the contract, such as the contract owner or the billing access controller. The [OCR2 contract](/contracts/ocr2/schema/query_msg.json) outlines a few possible string queries.
+
```

Comment:
```suggestion
Contract schemas for Terra can be found in the [contracts](/contracts) folder. Each contract in this directory has a subdirectory called **schema**. In these folders, you can learn how to query, execute, and instantiate each contract. 

## Queries

Queries are located in the **query_msg.json** files.

There are two types of queries: string queries and object queries. String queries are used to read a property of the contract, such as the contract owner or the billing access controller. The [OCR2 contract](/contracts/ocr2/schema/query_msg.json) outlines a few possible string queries.

```

Reply:
You can also add an `### Queries Via Gauntlet ` section, but leave it empty as not currently implemented, or add a simple comment that this is potentially incoming. 

And also an `## Gauntlet Inspect`  to the end where you could document inspection commands and how to use them. Feel free to also leave this as a future section, and keep this PR focused on simple queries.